app/scripts/lib/router.js

```diff
@@ -319,24 +317,36 @@ function (
 
           // if the first view errors, the fail branch of the promise will be
           // followed. The view will navigate to `unexpected_error`, which will
-          // eventually find its way here. `_firstViewHasLoaded` will still be
+          // eventually find its way here. `_hasFirstViewRendered` will still be
           // false, so broker.afterLoaded will be called. See
           // https://github.com/mozilla/fxa-content-server/pull/2147#issuecomment-76155999
-          if (! self._firstViewHasLoaded) {
-            // afterLoaded lets the RP know when the first screen has been
-            // loaded. It does not expect a response, so no error handler
-            // is attached and the promise is not returned.
-            self.broker.afterLoaded();
-
-            // back is enabled after the first view is rendered or
-            // if the user re-starts the app.
-            self.storage.set('canGoBack', true);
-            self._firstViewHasLoaded = true;
+          if (! self._hasFirstViewRendered) {
+            self._afterFirstViewHasRendered();
           }
           self._checkForRefresh();
         });
     },
 
+    _hasFirstViewRendered: false,
+    _afterFirstViewHasRendered: function () {
+      var self = this;
+      self._hasFirstViewRendered = true;
+
+      // afterLoaded lets the relier know when the first screen has been
+      // loaded. It does not expect a response, so no error handler
+      // is attached and the promise is not returned.
+      self.broker.afterLoaded();
+
+      // `loaded` is used to determine how long until the
+      // first screen is rendered and the user can interact
+      // with FxA. Similar to window.onload, but FxA specific.
+      self.metrics.logEvent('loaded');
```

Comment:
this PR looks good! What's your feeling on this event name? in datadog it is going to be: `fxa.content.loaded`
Is it too generic? What if we add logging for `LOADED` event that fires for Fennec. Would it be confusing? 
is `screen.loaded` better (because we have other 'screen' events)? 
Let me know